src/main/java/bobby/BobbyException.java

```diff
@@ -0,0 +1,8 @@
+package bobby;
+
+public class BobbyException extends Exception {
+    public BobbyException(String errorMessage) {
+        super(errorMessage);
```

Comment:
I think you can turn this into a more customize file, instead of simply extends the Exception class